.appveyor.yml

```diff
@@ -89,6 +89,8 @@ install:
   # but they have a broken certificate (on Windows) and an HSTS policy that
   # forces usage of HTTPS, so we mirror our own copy.
   - ps: If($Env:compiler -eq 'msys2-mingw') {(new-object Net.WebClient).DownloadFile('https://nirbheek.in/files/meson/get-pip.py', 'C:\projects\meson\get-pip.py')}
+  # pkg-config is needed for some msvc and cygwin tests
```

Comment:
It's misleading to mention cygwin in this comment.  We're not installing or using this pkg-config for cygwin, nor should we be.

Reply:
Hmm, I think I had to download this for cygwin too. Will try without it.

Reply:
I hope the condition `If($Env:compiler -ne 'msys2-mingw')` means this line isn't doing anything on cygwin, currently.

Reply:
I've changed it; it only installs for MSVC now.